pmm-app/src/pmm-dbaas/DBaaS.messages.ts

```diff
@@ -11,7 +11,7 @@ export const Messages = {
     deleteModal: {
```

Comment:
Create `deleteModal` section inside `dbcluster` section and use it in `DeleteDBClusterModal.tsx`, because this is being used in two different places and it shouldn't.

Reply:
Thanks, created separate fields